### PR TITLE
Fix/dg typos

### PR DESCRIPTION
**Pull request by jianrong7**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -635,14 +635,14 @@ System: PropertyMatch address book
 
 Actor: Property Agent
 
-1. Property agent identifies the specific criteria of properties or customerhe wants.
+1. Property agent identifies the specific criteria of properties or customer wants.
```
Comment (saraozn) [inline docs/DeveloperGuide.md:638]:
```suggestion
1. Property agent identifies the specific criteria of properties or customer he wants.
```

**Review by saraozn — DISMISSED** [review]
oop thanks, just a small change

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (jianrong7) [reply docs/DeveloperGuide.md:638]: xiexie

**Comment by jianrong7** [comment]
pls review @saraozn

**Review by nicolengk — APPROVED** [review]
😸

**Pull request merged**